src/patternfly/components/Dropdown/examples/dropdown-dropup-example.hbs

```diff
@@ -1,2 +1,8 @@
-{{#> dropdown id="dropdown-example-dropup-collapsed" pf-c-dropdown--Modifier="pf-m-dropup"}}Dropup{{/dropdown}}
-{{#> dropdown id="dropdown-example-dropup-expanded" pf-c-dropdown--IsExpanded="true" pf-c-dropdown--Modifier="pf-m-dropup"}}Dropup{{/dropdown}}
+{{#> dropdown id="dropdown-example-dropup-collapsed" pf-c-dropdown--Modifier="pf-m-dropup"}}
+  Dropup
+    <i class="fas fa-caret-down pf-c-dropdown__toggle-icon"></i> 
```

Comment:
Should this be defined in a handlebar so that we have one place where it's defined?

`<i class="fas fa-caret-down pf-c-dropdown__toggle-icon"></i>`